src/bundle.js

```diff
@@ -152,3 +153,26 @@ var headroom = require("headroom.js/dist/headroom");
 _define("headroom", function () {
     return headroom;
 });
+
+// apiclient
+var apiclient = require("jellyfin-apiclient");
+
+_define("apiclient", function () {
+    return apiclient.apiclient;
+})
+
+_define("events", function () {
+    return apiclient.Events;
+});
+
+_define("credentialprovider", function () {
+    return apiclient.Credentials;
+});
+
+_define("connectionManagerFactory", function () {
+    return apiclient.ConnectionManager;
+});
+
+_define("appStorage", function () {
+    return apiclient.AppStorage;
+});
```

Comment:
why are those needed, why cannot we just use `apiclient.Events` instead of `events`?

Reply:
It's (for now) to do as few code modifications as needed.